### PR TITLE
Migrate remaining scriptworkers

### PR DESCRIPTION
**Pull request by garbas**
- addonscript
- bouncerscript
- pushsnapscript
- pushapkscript
- shipitscript
- signingscript
- treescript

### CODE REVIEW

File: shipitscript/docker.d/init_worker.sh [hunk 1]
```diff
@@ -1,5 +1,14 @@
 #!/bin/bash
-set -e
+set -o errexit -o pipefail
+
+test_var_set() {
```
Comment (escapewindow) [inline shipitscript/docker.d/init_worker.sh:4]: Hm, should we define this once in `init.sh` and use that? I'm under the impression init.sh calls init_worker.sh, so we have access to the functions in both files.

File: signingscript/Dockerfile [hunk 2]
```diff
@@ -15,6 +15,7 @@ USER app
 WORKDIR /app
 
 RUN python -m venv /app \
+ && sed -i -e "s@ | base64 -d@@g" /app/docker.d/init.sh \
```
Comment (escapewindow) [inline signingscript/Dockerfile:18]: clever :)

File: .taskcluster.yml [hunk 3]
```diff
@@ -144,7 +164,7 @@ tasks:
                     cp ../docker.d/* docker.d/ &&
                     ./docker.d/generate_version_json.sh &&
                     ./docker.d/build_image.sh /tmp/image.tar &&
-                    if [ "$push_docker_image" == "1" ]; then
+                    if [ "$PUSH_DOCKER_IMAGE" == "1" ]; then
```
Comment (MihaiTabara) [inline .taskcluster.yml:167]: Thanks for fixing this!

**Review by escapewindow — APPROVED** [review]
Thanks!

File: signingscript/Dockerfile (hunk 2 above)
Comment (garbas) [reply signingscript/Dockerfile:18]: its just a temporary hack. you learn this with Nix :)

File: shipitscript/docker.d/init_worker.sh (hunk 1 above)
Comment (garbas) [reply shipitscript/docker.d/init_worker.sh:4]: i'm not sure, but i will check

**Review by MihaiTabara — APPROVED** [review]
We still need to address the `init.sh` special case for signingscript while my temporary hack is still in there.

File: signingscript/Dockerfile (hunk 2 above)
Comment (MihaiTabara) [reply signingscript/Dockerfile:18]: Ah, never mind my comment below, I didn't refresh before commenting :)

**Pull request merged**